packages/regenerator-runtime/runtime.js

```diff
@@ -85,7 +85,13 @@ var runtime = (function (exports) {
   function ensureDefaultToStringTag(object, defaultValue) {
     // https://bugzilla.mozilla.org/show_bug.cgi?id=1644581#c6
     return toStringTagSymbol in object
-      ? object[toStringTagSymbol]
+      ? (Object.defineProperty(object, toStringTagSymbol, {
```

Comment:
Should the condition maybe just be whether `Object.defineProperty` exists, since we're setting the tag either way, regardless of whether `toStringTagSymbol in object`?